tests/mock/test_agg.py

```diff
@@ -110,7 +111,7 @@ def mock_approx_percentile(
     ) -> ColumnEmulator:
         assert column.tolist() == [10.0, 20.0, 25.0, 30.0]
         assert percentile == 0.5
-        return ColumnEmulator(data=123)
+        return ColumnEmulator(data=123, sf_type=ColumnType(DoubleType(), False))
```

Comment:
Let's not enforce non-nullable unless we are asserting it.

Reply:
ColumnType is a namedtuple requiring the positional argument, I can flip it to true but I don't think it makes a different here..